pkg/operators/ebpf/extrainfo.go

```diff
@@ -109,6 +110,25 @@ func (i *ebpfInstance) addExtraInfo(gadgetCtx operators.GadgetContext) error {
 		Content:     []byte(variablesJson),
 	}
 
+	// add mermaid graph data
+	flowchartGraph, err := graphutils.GenerateFlowchartMermaidGraph(i.collectionSpec)
+	if err != nil {
+		return fmt.Errorf("generating mermaid flow graph: %w", err)
+	}
+	sequenceDiagram, err := graphutils.GenerateSequenceMermaidGraph(i.collectionSpec)
+	if err != nil {
+		return fmt.Errorf("generating mermaid sequence diagram: %w", err)
+	}
+
+	ebpfInfo.Data["ebpf.mermaid.flowchart"] = &api.GadgetInspectAddendum{
```

Comment:
I suggest dropping the "mermaid" from the name and keeping it simply "ebpf.flowchart" and "ebpf.sequence". The "mermaid" is just the type and, as such, is already in the `ContentType` field. WDYT?